testsuites/basic/lsvmbus.py

```diff
@@ -0,0 +1,82 @@
+import math
+
+import asserts
+
+from lisa import TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.environment import EnvironmentStatus
+from lisa.features import SerialConsole
+from lisa.testsuite import simple_requirement
+from lisa.tools import Lscpu, Lsvmbus
+
+
+class VmbusNames:
+    def __init__(self, is_gen1: bool) -> None:
+        self.vmbus_names = [
+            "Operating system shutdown",
+            "Time Synchronization",
+            "Heartbeat",
+            "Data Exchange",
+            "Synthetic mouse",
+            "Synthetic keyboard",
+            "Synthetic network adapter",
+            "Synthetic SCSI Controller",
+        ]
+        if is_gen1:
+            self.vmbus_names.append("Synthetic IDE Controller")
+
+
+@TestSuiteMetadata(
+    area="core",
+    category="functional",
+    description="""
+    This test suite uses to check correctness of vmbus channel
+    """,
+    tags=[],
+)
+class LsVmBus(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This test is to check channel count of each netvsc
+        and storvsc SCSI device.
+        When count is unexpected, then test fail.
+        """,
+        priority=1,
+        requirement=simple_requirement(
+            environment_status=EnvironmentStatus.Deployed,
+            supported_features=[SerialConsole],
+        ),
+    )
+    def lsvmbus_test(self) -> None:
```

Comment:
The case name can be `lsvmbus_channel_counting`, since we may create multiple lsvmbus test cases.